modules/calib3d/include/opencv2/calib3d.hpp

```diff
@@ -841,6 +841,34 @@ square grouping and ordering algorithm fails.
 CV_EXPORTS_W bool findChessboardCorners( InputArray image, Size patternSize, OutputArray corners,
                                          int flags = CALIB_CB_ADAPTIVE_THRESH + CALIB_CB_NORMALIZE_IMAGE );
 
+/** @brief Finds the positions of internal corners of the chessboard using a sector based approach.
+
+@param image Source chessboard view. It must be an 8-bit grayscale or color image.
+@param patternSize Number of inner corners per a chessboard row and column
+( patternSize = cvSize(points_per_row,points_per_colum) = cvSize(columns,rows) ).
```

Comment:
`Size` instead of `cvSize`?

Reply:
Fair, copy paste error from findChessboardCorners. I would propose to change it as well:
https://github.com/opencv/opencv/blob/053259fd92c1736d3920c7d43cb0400c518c4a5e/modules/calib3d/include/opencv2/calib3d.hpp#L795